src/sections/CareersHappiness/index.css

```diff
@@ -0,0 +1,94 @@
+@import "../../styles/all.css";
+
+.root {
+  display: flex;
+  padding-bottom: $spacing-224;
+
+  @include Breakpoint-mobileOnly {
+    display: block;
+
+    padding-bottom: $spacing-136;
+  }
+}
+
+.image {
+  height: $spacing-584;
+
+  @include Breakpoint-tabletOnly {
+    height: $spacing-352;
+  }
+}
+
+.container-left {
+  width: $six-columns;
+
+  @include Breakpoint-desktopOnly {
+    margin-right: $column-spacing;
+  }
+  @include Breakpoint-tabletOnly {
+    width: 60%;
+    margin-right: $column;
+  }
+
+  .image {
+    margin-left: calc(-1 * $column-spacing);
```

Comment:
not 100% sure but I think we don't need calc for negative spacing

Reply:
postcss return an error with `-$column-spacing`, but I don't need calc to `-$gutter` as I used to use.